backend/src/main/java/com/znane_groby/backend/controller/CityResourceAssembler.java

```diff
@@ -0,0 +1,22 @@
+package com.znane_groby.backend.controller;
+
+import com.znane_groby.backend.model.City;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.server.RepresentationModelAssembler;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Component
+class CityResourceAssembler  implements RepresentationModelAssembler<City, EntityModel<City>> {
```

Comment:
To te rzeczy co ja ich nie umiem, więc nie komentuję. Myślę że jeśli zaczniemy mieć z tym problem to możemy olać, bo nie wiem czy wykorzystamy te linki, a w sumie robimy bardziej api niż zasób, ale to na marginesie.

Reply:
kiedys wspominaleś, że możemy też otworzyć tego RESTa dla innych i w sumie to linkowanie wyglądało fajnie (ten assembler tylko trochę to usprawnia). Nie wiem tylko czemu pluje mi się w Ideii jako depraceded (jeden fragment). No i chwilę zajmuje zabawa z tym, ale podobno (tak twierdził jakiś tutorial springowy) tak jest bardziej RESTOwo, ale jeśli nikt tego nie używa (nie licząc tego jednego tutoriala) to bym się nie upierał